interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -293,6 +304,10 @@ namespace cling {
 
     llvm::LLVMContext* getLLVMContext() { return m_LLVMContext.get(); }
 
+    IncrementalExecutor* getIncrementalExecutor() { return m_Executor.get(); }
+
+    void setExternalIncrementalExecutor(IncrementalExecutor* extIncr);
```

Comment:
What do we need this getter and setter for? They seem to be unused, and I don't see a need for them.